meta-ostro/recipes-kernel/bcm43340/bcm43340-mod.bbappend

```diff
@@ -0,0 +1 @@
+COMPATIBLE_MACHINE="edison"
```

Comment:
On Thu, Mar 31, 2016 at 2:44 AM, Alexander D. Kanevskiy <
notifications@github.com> wrote:

> In meta-ostro/recipes-kernel/bcm43340/bcm43340-mod.bbappend
> <https://github.com/ostroproject/meta-ostro/pull/63#discussion_r57982143>:
>
> > @@ -0,0 +1 @@
> > +COMPATIBLE_MACHINE="edison"
>
> should those be here ?
>
>
No. Just to test build/testing purposes.